src/plugins/intel_gpu/src/graph/impls/ocl/permute.cpp

```diff
@@ -47,7 +47,10 @@ struct permute_impl : typed_primitive_impl_ocl<permute> {
     DECLARE_OBJECT_TYPE_SERIALIZATION(cldnn::ocl::permute_impl)
 
     std::unique_ptr<primitive_impl> clone() const override {
-        return make_unique<permute_impl>(*this);
+        auto prim_impl = make_unique<permute_impl>(*this);
+        kernel_params_t* params_ptr = dynamic_cast<kernel_params_t*>(prim_impl->_kernel_data.params.get());
+        prim_impl->_kernel_data.params = make_unique<kernel_params_t>(*params_ptr);
+        return prim_impl;
```

Comment:
I think we can have a template helper to minimize code copy-paste.

Reply:
@vladimir-paramuzov Addressed with commit 0ccef8f.

Reply:
I think it can embed make_unique part as well.
```
std::unique_ptr<primitive_impl> clone() const override {
    return make_deep_copy<permute_impl, kernel_params_t>(*this);
}
```

Reply:
@vladimir-paramuzov embed all with commit af3efb4.